governance/votes/Trial Executive vote - Aug 2, 2023.md

```diff
@@ -0,0 +1,142 @@
+
+# [Executive Proposal] DSR Increase, Spark D3M Changes, Harbor Trade Credit (RWA004-A) Changes, New Silver (RWA002-A) Changes, Launch Project Funding, AVC Member Participation Rewards, Clydesdale Quarterly Surplus Return - August 2, 2023
+
+The Governance Facilitator(s), AmusingAxl, Sidestream and Arby have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+- Increase the Dai Savings Rate (DSR) by 4.81% from 3.19% to 8%.
+- Increase the DIRECT-SPARK-DAI Maximum Debt Ceiling by 180 million DAI from 20 million DAI to 200 million DAI, as described below.
+- Set Spark Protocol DAI market Interest Strategy to track the ETH-C stability fee instead of the DSR rate as described below.
+- Set Spark Protocol DAI market Loan-To-Value (LTV) and Liquidation Threshold to 0.01% as described below.
+- Set Spark Protcol WETH market Reserve Factor to 5% as described below.
+- Set Spark Protocol WETH market Variable Rate Slope 1 to 3% as described below.
+- Decrease Harbor Trade Credit Vault (RWA004-A) Debt Ceiling as detailed below.
+- Increase New Silver (RWA002-A) Debt Ceiling and Stability Fee as described below.
+- A MakerDAO Resolution will be approved to initiate a Clydesdale quarterly surplus return as described below.
+- Distribute **2,000,000 DAI** for Launch Project funding as described below.
+- Distribute **134.1 MKR** for Q2 AVC Member Participation Rewards as described below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Dai Savings Rate Increase
+
+In alignment with the recently approved [Enhanced Dai Savings Rate (EDSR)](https://vote.makerdao.com/polling/QmcTRPLx#poll-detail) system, the following changes to the [Dai Savings Rate (DSR)](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) will be made:
```

Comment:
Important(?) to use the terminology: 'will be made if this executive proposal passes'

Reply:
We've always seen it as important anyway. Makes it clear that governance is the party that is triggering the changes.